### ISSUE TITLE
How To Reach Parent Folder?

### ISSUE THREAD

**Issue by BayuAngora** [11ty/eleventy#1215]
I have this nested section -> 
```
content/
    |_ blog/
        |_ index.md
    |_ poem/
        |_ 2020
            |_ 2020.json
            |_ new-poem.md
```
When I'm on domain.tld/new-poem/ page, how to reach parent folder "poem" and "2020"?

**Comment by dtroode** [comment]
I think you should add an `_index.md` into `_2020` directory (or any other index, depends on you template language).

Or another solution is to add a `_2020.md` into `_poem` directory. Works for me.

**Comment by BayuAngora** [comment]
> I think you should add an `_index.md` into `_2020` directory (or any other index, depends on you template language). This will add an access to /2020 path. The same for `_poem`.
> 
> Or another solution is to add a `_2020.md` into `_poem` directory and `_poem.md` into `content` directory. Works for me.

Yes, it's reachable with that method. But I mean, how to reach them via parameter?

So, if I make a button from domain.tld/new-poem/ page, I can reach domain.tld/poem/ and domain.tld/poem/2020/ 

I try to add ```{{ parent }}``` and ```{{ parent.url }}``` but it doesn't works?

**Comment by pdehaan** [comment]
@BayuAngora I must not be understanding... 
If you're on "domain.tld/new-poem/" and you wanted to reach "domain.tld" poem, wouldnt you just use `<a href="/poem/">something</a>`?

Not sure where you're trying to access `{{ parent }}`, if it's in an `href` or elsewhere.

**Comment by BayuAngora** [comment]
> @BayuAngora I must not be understanding...
> If you're on "domain.tld/new-poem/" and you wanted to reach "domain.tld" poem, wouldnt you just use `<a href="/poem/">something</a>`?
> 
> Not sure where you're trying to access `{{ parent }}`, if it's in an `href` or elsewhere.

Yes, I can use that to reach "/poem/" folder. But if I want to reach "/poem/2010/" and "/poem/2011/" and "/poem/2012/" and more, I have to implement it one by one. 

With Hugo, I can implement it by ```{{ .Parent.Title }}``` and ```{{ .Parent.URL }}```. So, everytime I post a new poem in every year folder, it can be detected and sorted automatically. 

I don't know how to implement it with Eleventy?

**Comment by simplerethink** [comment]
Has anyone figured this out? I'm trying to do something similar. The Hugo example is exactly what I'm trying to accomplish as well.

**Comment by BayuAngora** [comment]
> 

On Hugo I can implement it with this code:
```
<a href="{{ .Parent.Permalink }}">
<div class="center">
{{ .Parent.Title }}
</div>
</a>
```
I still curious how to implement it on 11ty.

**Comment by zachleat** [comment]
Lots of overlap with #3794 here but this makes me think it might be more useful to have a supplied universal filter to get the parent directory of a path or URL.
